src/app/sessions/components/sessions-amend-form/sessions-amend-form.component.html

```diff
@@ -38,6 +38,10 @@
       <div>
         <span>Number of listing requests: <strong>{{amendSessionForm.hearingPartCount}}</strong></span>
       </div>
+
+      <div *ngIf="amendSessionForm.belongsToMultiSession">
+        <span style="font-style: italic">This session contains a multi-session hearing</span>
```

Comment:
Please move to css as a class, probably could be put into global styles and be reused later on in different places.